.phan/stubs/activitypub.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Phan stubs for the ActivityPub plugin.
+ * https://github.com/Automattic/wordpress-activitypub/
+ *
+ * @package automattic/jetpack-monorepo
+ */
+
+namespace Activitypub;
+
+/**
+ * Stub for is_activitypub_request function.
+ *
+ * @return bool
+ */
+function is_activitypub_request() {
+	return false;
+}
```

Comment:
As long as the signature and phpdoc are correct so Phan can know the right param and return types, the body is irrelevant and there's no point in making Phan parse it.
```suggestion
function is_activitypub_request() {
}
```